src/games/googlegossip/GoogleGossip.js

```diff
@@ -190,6 +190,7 @@ export default class GoogleGossip extends GameComponent {
           <br />
           
           <input
+            className="gg-input"
```

Comment:
I already scoped these elements in [another commit]. This explains the merge conflicts you're seeing. Mind reverting this changes and rebasing to avoid the conflict?

[another commit]:https://github.com/jtessler/studio-multiplayer-game/commit/e6b793771eaf1f630e7c4a1969885351ef0045a6